javascript/README.md

```diff
@@ -0,0 +1,13 @@
+# Building the JavaScript libsbp bindings
+
+From the parent directory, run `make javascript` to regenerate bindings.
+
+# Using the JavaScript libsbp bindings
```

Comment:
Not really related to this readme, but can we package the javascript client as well? npm or something?